packages/main/src/plugin/docker-extension/docker-plugin-adapter.ts

```diff
@@ -84,7 +84,7 @@ export class DockerPluginAdapter {
             updatedArgs = args;
           }
 
-          const spawnProcess = spawn(updatedCommand, updatedArgs, { env });
+          const spawnProcess = spawn(updatedCommand, updatedArgs, { env, shell: true });
```

Comment:
As the arguments are passed as an array, there is no need to escape them with spaces so is this necessary and not error prone ?

Reply:
the thing is that arguments are like `ddClient.docker.cli.exec("info", ["--format", "'{{json .}}'"])`

https://github.com/microcks/microcks-docker-desktop-extension/blob/927a51c91e7dcb5f5b40e4da4f98044cba2cb1d0/client/src/api/metrics.ts#L44

so they are providing `"'{{json .}}'"` which is then interpreted by default as `'{{json .}}'` and return `'{content of the json}'` instead of `{content of the json}`

Reply:
I would guess that they should pass "{{json .}}" but maybe that will cause issues with Docker Desktop then

Reply:
yes, it's to be bug compatible